app/models/spree/variant.rb

```diff
@@ -142,8 +144,17 @@ class Variant < ApplicationRecord
     scope :stockable_by, lambda { |enterprise|
       return where("1=0") if enterprise.blank?
 
-      joins(:product).
-        where(spree_products: { id: Spree::Product.stockable_by(enterprise).pluck(:id) })
```

Comment:
I think `Spree::Product.stockable_by` is now no longer used and can be removed.

Reply:
Indeed, I removed it.